Taskfile.yml

```diff
@@ -15,14 +15,14 @@ tasks:
   deps:
     desc: Install dependencies
     cmds:
-      - curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.54.2
-      - go install github.com/princjef/gomarkdoc/cmd/gomarkdoc@v1.1.0
-      - go install golang.org/x/vuln/cmd/govulncheck@v1.0.1
-      - go install github.com/jstemmer/go-junit-report@v0.9.1
-      - go install github.com/segmentio/golines@v0.11.0
-      - go install github.com/golang/mock/mockgen@v1.6.0
-      - go install github.com/boumenot/gocover-cobertura@v1.2.0
-      - go install mvdan.cc/gofumpt@fd93f1d8818fc86a956b9706ad8b81734c81cc1e
+      - curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin latest
```

Comment:
They seem to have gotten whatever made this workaround necessary sorted out, so
```
go install github.com/golangci/golangci-lint/cmd/golangci-lint@latest
```
...appears to work as expected.